test/Test/Fission/DNS.hs

```diff
@@ -13,15 +16,13 @@ tests =
         itsProp' "length does not change" \(SmallText txt) -> 
           DNS.splitRecord txt `shouldBe` pure txt
 
-      -- @@TODO: Add this test back in once the LargeText Arbitrary instance works
-
-      -- context "over 256 characters" do
-      --   itsProp "length does not change" 1 \(LargeText txt) -> 
-      --     splitCount txt `shouldBe` expectedCount txt
+      context "over 256 characters" do
+        itsProp' "length does not change" \(LargeText txt) -> 
+          splitCount txt `shouldBe` expectedCount txt
 
-      --     where
-      --       splitCount = NonEmpty.length . DNS.splitRecord
-      --       expectedCount txt = (Text.length txt `div` 252) + 1
+          where
+            splitCount = NonEmpty.length . DNS.splitRecord
```

Comment:
Yep looking good! Let’s also add a prop test that it serializes and deserializes correctly (I’m assuming that this should be trivial — if not just make a note)

Reply:
i.e.

```haskell
fromDNS (forDNS txt) `shouldBe` txt
```

Reply:
But yeah, I’ll let you use your judgement on that. LGTM!

Reply:
we don't have a function for recombining records yet (I think that functionality is only needed in the SDK at the moment). so I'll hold off on that & add a note for when/if we add it